devito/core/gpu_openmp.py

```diff
@@ -14,6 +14,16 @@
                                ParallelTree, optimize_halospots, mpiize, hoist_prodders,
                                iet_pass)
 from devito.tools import as_tuple, filter_sorted, timed_pass
+##
```

Comment:
all of the pep8 stuff/ flake8 can be fixed locally very quickly... inspecting through:
`flake8 ....changed files...```